setup/pi/create-backingfiles-partition.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/bash -eu
 
+HEADLESS_SETUP=${HEADLESS_SETUP:-false}
+
+function setup_progress () {
+  SETUP_LOGFILE=/boot/teslausb-headless-setup.log
```

Comment:
Does this need to be global? If so, I'd like to see it declared and initialized outside the function with the rest of the globals. If not, I'd like to see it declared local.

Reply:
It's resolved for now by putting it local into the function. (Pushed.) 

TBH I'm not 100% adept at tracking the globals, or exporting functions, when multiple script depths come into play at this point, so there's probably a cleaner way to do it all up front. I'd like to explore that with your guidance, and minimize the logic in rc.local even further, but it'll take a bit so hopefully this resolves for now (and gives pointers on what needs to be optimized later).